### PR TITLE
SAK-48171 - Gradebook: Cell popovers fail to display for new items

### PR DESCRIPTION
**Pull request by hornersa**
A screenshot, ProposedPopoverFix.png, of the proposed fix is attached to https://sakaiproject.atlassian.net/browse/SAK-48171.

### CODE REVIEW

File: gradebookng/tool/src/webapp/scripts/gradebook-gbgrade-table.js [hunk 1]
```diff
@@ -3451,7 +3452,8 @@ GbGradeTable.focusColumnForAssignmentId = function(assignmentId, showPopupForNew
                   });
               }
             })
-            (new bootstrap.Popover($selectedField[0])).show();
+            var p = new bootstrap.Popover($selectedField[0]);
+            p.show();
```
Comment (adrianfish) [inline gradebookng/tool/src/webapp/scripts/gradebook-gbgrade-table.js:3456]: Why are you adding this p var here? The single line should work ok

**Review by adrianfish — CHANGES_REQUESTED** [review]
I don't think you need those two lines to show the popover. Is there a reason you changed that? Does it cause a problem?

**Comment by hornersa** [comment]
While I concur that the single line seems like it should work, splitting that line was intentional to avoid the following error (in Chrome), which prevents the popover from displaying: 

gradebook-gbgrade-table.js?version=53e598a3:3455 Uncaught TypeError: $(...).on(...) is not a function
    at gradebook-gbgrade-table.js?version=53e598a3:3455:13
    at Function.<anonymous> (gradebook-gbgrade-table.js?version=53e598a3:3414:13)
    at Function.each (jquery.js:370:19)
    at GbGradeTable.runReadyCallbacks (gradebook-gbgrade-table.js?version=53e598a3:3412:7)
    at GbGradeTable.renderTable (gradebook-gbgrade-table.js?version=53e598a3:1327:16)
    at HTMLDocument.<anonymous> (grades?focusAssignmentId=121&newItem=true:156:14)
    at i (jquery.js:3232:31)
    at Object.fireWith [as resolveWith] (jquery.js:3362:7)
    at Function.ready (jquery.js:3582:13)
    at HTMLDocument.K (jquery.js:3617:10)

File: gradebookng/tool/src/webapp/scripts/gradebook-gbgrade-table.js (hunk 1 above)
Comment (adrianfish) [inline gradebookng/tool/src/webapp/scripts/gradebook-gbgrade-table.js:3456]:
```suggestion
            new bootstrap.Popover($selectedField[0]).show();
```

Comment (adrianfish) [reply gradebookng/tool/src/webapp/scripts/gradebook-gbgrade-table.js:3456]: That works. I just tried it on your branch.

Comment (hornersa) [reply gradebookng/tool/src/webapp/scripts/gradebook-gbgrade-table.js:3456]: Agreed. I'll force push an edit momentarily.

**Review by adrianfish — APPROVED** [review]

**Pull request merged**